app/thirdparty/java/other/writeily/model/WrMarkorSingleton.java

```diff
@@ -12,17 +12,13 @@
 import android.app.Activity;
 import android.content.Context;
 import android.support.v4.provider.DocumentFile;
-import android.widget.Toast;
 
 import net.gsantner.markor.format.TextFormat;
 import net.gsantner.markor.ui.SearchOrCustomTextDialogCreator;
 import net.gsantner.markor.util.ShareUtil;
-
-import org.apache.commons.io.IOUtils;
+import net.gsantner.opoc.util.FileUtils;
```

Comment:
@gsantner Is using opoc here ok?

Reply:
Sure. No problem opoc can be used anywhere